src/chocolatey.tests/infrastructure.app/commands/ChocolateyInfoCommandSpecs.cs

```diff
@@ -134,6 +134,61 @@ public void should_add_short_version_of_password_to_the_option_set()
             }
         }
 
+        public class when_handling_validation : ChocolateyInfoCommandSpecsBase
+        {
+            private Exception error = null;
+            private Action because;
+
+            public override void Context()
+            {
+                base.Context();
+            }
+
+            public override void Because()
+            {
+                because = () => command.handle_validation(configuration);
+            }
+
+            [Fact]
+            public void show_throw_when_package_id_is_not_set()
+            {
+                configuration.Input = "";
+                error = null;
+
+                try
+                {
+                    command.handle_validation(configuration);

```

Comment:
```suggestion
                    because();
```

The test we initially copied from used the `because` Action to perform the setup in the tests. We should either do the same or remove the `because` Action.